crates/sui-graphql-rpc/examples/owner/dynamic_field.graphql

```diff
@@ -0,0 +1,51 @@
+fragment DynamicFieldValueSelection on DynamicFieldValue {
+  ... on MoveValue {
+    type {
+      repr
+    }
+    data
+    __typename
+  }
+  ... on MoveObject {
+    hasPublicTransfer
+    contents {
+      type {
+        repr
+      }
+      data
+    }
+    __typename
+  }
+}
+
+fragment DynamicFieldNameSelection on MoveValue {
+  type {
+    repr
+  }
+  data
+  bcs
+}
+
+fragment DynamicFieldSelect on DynamicField {
+  name {
+    ...DynamicFieldNameSelection
+  }
+  value {
+    ...DynamicFieldValueSelection
+  }
+}
+
+query DynamicField {
+  object(
+    address: "0xb57fba584a700a5bcb40991e1b2e6bf68b0f3896d767a0da92e69de73de226ac"
+  ) {
+    dynamicField(
+      dynamicFieldName: {
```

Comment:
nit: Seeing this all written out, probably it's best to just turn this into `name`? What do you think?

```suggestion
      name: {
```

We would need to update the draft schema too.

Reply:
sounds good to me, lets shorten it to name